src/extensions/extensionsHelper.ts

```diff
@@ -405,6 +411,7 @@ export async function publishExtensionVersionFromLocalSource(args: {
   rootDirectory: string;
   nonInteractive: boolean;
   force: boolean;
+  stage: "rc" | "alpha" | "beta" | "stable";
```

Comment:
Also, it would be good to declare this type outside of this function, since we may need to reference it elsewhere.